### PR TITLE
Fix closing/opening state for awning

### PR DESCRIPTION
**Pull request by tetienne**
Fix https://github.com/iMicknl/ha-tahoma/issues/730

### CODE REVIEW

File: custom_components/tahoma/cover.py [hunk 1]
```diff
@@ -32,10 +32,9 @@ async def async_setup_entry(
     ]
 
     entities += [
-        VerticalCover(device.device_url, data.coordinator, low_speed=True)
+        LowSpeedCover(device.device_url, data.coordinator)
```
Comment (tetienne) [inline custom_components/tahoma/cover.py:35]: I’ve also backported this.

**Comment by tetienne** [comment]
@iMicknl Can you review this please?

**Comment by iMicknl** [comment]
> @iMicknl Can you review this please?

Is this 1:1 core migration?

**Comment by tetienne** [comment]
@tetienne Nope, do you want I open the same PR within Core also before we merge this one?

**Pull request closed without merging**